src/overview/actions.js

```diff
@@ -9,6 +9,8 @@ import { ourState } from './selectors'
 
 export const setQuery = createAction('overview/setQuery')
 export const setSearchResult = createAction('overview/setSearchResult')
+export const incrementLoadCounter = createAction('overview/incrementLoadCounter')
+export const decrementLoadCounter = createAction('overview/decrementLoadCounter')
```

Comment:
Hmm.. I'm thinking that the fact that they count the amount of running searches could be considered an implementation detail of the reducer logic. The actions could perhaps just be `show`/`hideLoadingIndicator`, which expresses their intention, not their implementation. It may be slightly confusing that a hide-action would in some cases not actually hide the indicator, but that seems acceptable to me.